recipes/foundry-compute-modules/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "foundry-compute-modules" %} 
+{% set version = "0.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/palantir/python-compute-module/archive/{{ version }}.tar.gz
+  sha256: 89002198e1a9fca89130ece82c51e8a49f914c58f78ddf67eb18cac1e977fcee
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core
+    - pip
+  run:
+    - python >={{ python_min }}
+    - requests >=2.32.3
+
+test:
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+  imports:
+    - compute_modules
+
+about:
+  home: https://github.com/palantir/python-compute-module
+  summary: The official Python library for creating Compute Modules
+  license: Apache-2.0
+  license_family: Apache
+  # See https://github.com/palantir/python-compute-module/blob/develop/LICENSE.txt
+  license_file: LICENSE.txt
+  doc_url: https://www.palantir.com/docs/foundry/compute-modules/overview
+  dev_url: https://github.com/palantir/python-compute-module
+
+
+extra:
+  recipe-maintainers:
+    # This is maintained by Palantir Technologies, but you cannot list an organization as the recipe-maintainers
```

Comment:
```suggestion
```
recipes on conda-forge are maintained by individuals. The comment is misleading. Please remove it.